### PR TITLE
Lps 183998 | Add tests for story LPS-161184 - Generate export endpoints automatically for headless APIs

### PR DESCRIPTION
**Pull request by yanan**
Background
- Add automation tests for story LPS-161184 Generate export endpoints for headless APIs

Jira Ticket
- https://issues.liferay.com/browse/LPS-183998

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
Failures are unrelated to the pr and will be fixed in https://github.com/liferay-headless/liferay-portal/pull/1233

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro [hunk 1]
```diff
@@ -769,6 +769,20 @@ definition {
...
+		var portalURL = JSONCompany.getPortalURL();
+
+		var curl = '''
+			${portalURL}/o/object-admin/v1.0/object-definitions/export-batch \
+				-H Content-Type: application/json \
+				-u test@liferay.com:test
+		''';
+
+		var exportBatchId = JSONCurlUtil.post(${curl}, "$.id");
+
```
[the remark this answers is not in the document]
Comment (yanan) [reply modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:782]: you mean update to objectDefinitionsExportBatchId here? No problem, I will see if any other chagnes required from Meg and updated them together.

[the remark this answers is not in the document]
Comment (magjed4289) [reply modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:782]: yeah, that's a good idea to differentiate it here

**Review by magjed4289 — APPROVED** [review]
no new changes from my side. added some comments to what Kevin proposed.

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:forward

**Pull request closed without merging**